org.kiwix.desktop.json

```diff
@@ -0,0 +1,121 @@
+        "/bin/xapian-*",
+        "/share/aclocal",
+        "/share/doc",
+        "/share/man"
+    ],
+    "modules": [
+        {
+            "name": "pugixml",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "archive",
+                    "sha256": "0f422dad86da0a2e56a37fb2a88376aae6e931f22cc8b956978460c9db06136b",
+                    "url": "http://download.kiwix.org/dev/pugixml-1.2.tar.gz",
+                    "dest": "src"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/pugixml_meson.patch"
+                }
+            ]
+        },
+        {
+            "name": "xapian-core",
+            "config-opts": [
+                "--disable-sse",
+                "--disable-backend-chert",
+                "--disable-backend-inmemory",
+                "--disable-documentation"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "sha256": "13f08a0b649c7afa804fa0e85678d693fd6069dd394c9b9e7d41973d74a3b5d3",
+                    "url": "http://download.kiwix.org/dev/xapian-core-1.4.7.tar.xz"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/xapian_sys_types.patch"
+                }
+            ]
+        },
+        {
+            "name": "libzim",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/openzim/libzim.git",
+                    "tag": "4.0.4"
+                }
+            ]
+        },
+        {
+            "name": "kiwix-lib",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/kiwix/kiwix-lib.git",
+                    "tag": "3.1.1"
+                }
+            ]
+        },
+        {
+            "name": "aria2",
+            "config-opts": [
+                "--disable-libaria2",
+                "--disable-websocket",
+                "--without-sqlite3"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "sha256": "3a44a802631606e138a9e172a3e9f5bcbaac43ce2895c1d8e2b46f30487e77a3",
+                    "url": "https://github.com/aria2/aria2/releases/download/release-1.34.0/aria2-1.34.0.tar.xz"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/libaria2_android.patch"
+                }
+            ]
+        },
+        {
+            "name": "kiwix-desktop",
+            "buildsystem": "qmake",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/kiwix/kiwix-desktop.git",
+                    "tag": "2.0-beta2"
```

Comment:
Specific commits are preferred as tags can be modified to point to different commit.

Reply:
Changed in last forced push.